### PR TITLE
reviewed spanish expressions

### PR DESCRIPTION
**Pull request by mm-s**

### CODE REVIEW

File: locales/es.json [hunk 1]
```diff
@@ -2,9 +2,9 @@
 	"select-user-type.who_you_are": "¿Cual es su perfil? (Selecciona por ahora una opción, esto lo podrá cambiar posteriormente)",
 	"select-user-type.driver": "Conductor",
 	"select-user-type.passenger": "Pasajero",
-	"request-phone.type_your_phone": "Por favor envían tu número telefónico. No vamos a compartir tu información.",
+	"request-phone.type_your_phone": "Por favor envía tu número de telefóno. No vamos a compartir tu información.",
```
Comment (jbrizio) [inline locales/es.json]: The correct spelling is `teléfono`

File: locales/es.json [hunk 2]
```diff
@@ -14,42 +14,42 @@
 	"passenger-request-destination.provide_destination": "¿A donde desea ir? Por favor escriba la dirección exacta de la calle (Así el sistema GPS podrá entender) o describa el lugar.",
 	"order-submitted.order_submitted": "⏳ Localizando conductores... Si usted encontro un conductor, presione el botón de abajo 👇",
 	"order-submitted.cancel": "Terminar búsqueda",
-	"order-submitted.error_incorrect_input": "Opción desconocida, por favor intente de nuevo.",
+	"order-submitted.error_incorrect_input": "Opción desconocida, por favor intentelo de nuevo.",
```
Comment (jbrizio) [inline locales/es.json]: The same case here. The correct spelling is `inténtelo`

File: locales/es.json [hunk 3]
```diff
@@ -14,42 +14,42 @@
...
+	"driver-select-vehicle-type.select": "Seleccione el tipo de vehículo que desea pedir.",
 	"driver-select-vehicle-type.motorbike": "🏍  Motoneta / Motocicleta",
 	"driver-select-vehicle-type.car": "🚗 Automóvil",
 	"driver-select-vehicle-type.error_only_known_type": "Opción desconocida, por favor seleccione automóvil o motoneta / motocicleta.",
-	"driver-explain-checkins.text": "Casi terminamos, algunas cosas a tener en cuenta. Telegram no tiene soporte activo de localización GPS así que cada vez que cambie su ubicación realice un registro en la aplicación o registre cada hora si esta listo para aceptar nuevas ordenes de viaje. Esto puede ayudar a encontrar pasajeros cercanos a su ubicación. Presiona \"Siguiente\"",
+	"driver-explain-checkins.text": "Ya Casi hemos acabado, algunas cosas a tener en cuenta. Telegram no tiene soporte activo de localización GPS así que cada vez que cambie su ubicación realice un registro en la aplicación o registre cada hora si esta listo para aceptar nuevas ordenes de viaje. Esto puede ayudar a encontrar pasajeros cercanos a su ubicación. Presiona \"Siguiente\"",
```
Comment (jbrizio) [inline locales/es.json]: Delete the Uppercase of `Casi`

File: locales/es.json [hunk 4]
```diff
@@ -14,42 +14,42 @@
...
 	"driver-explain-checkins.next": "Siguiente",
-	"driver-request-location.provide_location": "Por favor comparta su ubicación actual, así los pasajeros cercanos a usted pueden encontrarle.",
-	"driver-explain-whats-next.text": "¡Estamos listos! Será notificado sobre pasajeros y ordenes cercanas a usted. Mantén esta aplicación activa y realice registros cada hora para actualizar su ubicación. Presione \"Siguiente\"",
+	"driver-request-location.provide_location": "Por favor comparta su ubicación actual, de modo que los pasajeros cercanos a usted puedan encontrarle.",
+	"driver-explain-whats-next.text": "¡listo! Será notificado sobre pasajeros y ordenes en su cercanos a usted. Mantenga esta aplicación activa y realice registros cada hora para actualizar su ubicación. Presione \"Siguiente\"",
```
Comment (jbrizio) [inline locales/es.json]: I guess that the correct phrase should be:
`¡Listo! Usted será notificado sobre pasajeros y ordenes cercanas.`

**Comment by ro31337** [comment]
@mm-s do you agree with @jbrizio comments? If so, please add those fixes, or leave a comment here. Thanks

**Comment by ro31337** [comment]
Also @asccigcc what do you think?

**Comment by asccigcc** [comment]
Agree, it is a typo. Good catch guys.

File: locales/es.json (hunk 4 above)
Comment (mm-s) [inline locales/es.json]: Yes, sorry, I didn't review my review well..

File: locales/es.json (hunk 2 above)
Comment (asccigcc) [inline locales/es.json]: Guys maybe we should decide if the translation should be usted or tu. Warhammer do you think @jbrizio?

Comment (jbrizio) [inline locales/es.json]: I prefer to use _"usted"_

**Comment by Dannymatos73** [comment]
my location is 730 Richmond ave san Jose CA 95128 #3

On Feb 3, 2017 7:13 AM, "Jonathan Brizio" <notifications@github.com> wrote:

*@jbrizio* commented on this pull request.
------------------------------

In locales/es.json <https://github.com/ro31337/libretaxi/pull/437>:

> @@ -14,42 +14,42 @@
 	"passenger-request-destination.provide_destination": "¿A donde desea
ir? Por favor escriba la dirección exacta de la calle (Así el sistema
GPS podrá entender) o describa el lugar.",
 	"order-submitted.order_submitted": "⏳ Localizando conductores... Si
usted encontro un conductor, presione el botón de abajo 👇",
 	"order-submitted.cancel": "Terminar búsqueda",
-	"order-submitted.error_incorrect_input": "Opción desconocida, por
favor intente de nuevo.",
+	"order-submitted.error_incorrect_input": "Opción desconocida, por
favor intentelo de nuevo.",

I prefer to use *"usted"*

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/ro31337/libretaxi/pull/437>, or mute the thread

**Comment by ro31337** [comment]
Merged to master, feel free to add more improvements here, if any.

**Comment by mm-s** [comment]
I've updated, there was a mixture of 'usted' and  'tu' usages, I've converted to 'tu' form as in Europe does not sound formal in excess, Latino America may have other preferences.

**Comment by asccigcc** [comment]
Probably we should create es_MX and es_ES for each region. What do you think guys?

**Pull request merged**